tests/UIButtons.js

```diff
@@ -0,0 +1,173 @@
+      console.log(fontSize);
+      const valueFontSize = await fontSize.value;
+      await expect(valueFontSize).toEqual("16px");
+
+      const color = await object.getCSSProperty("color");
+      console.log(color);
+      const valueColor = await color.parsed.hex;
+      await expect(valueColor).toEqual("#898989");
+
+      const pandding = await object.getCSSProperty("padding-right");
+      console.log(pandding);
+      const valuePandding = await pandding.value;
+      await expect(valuePandding).toEqual("10px");
+
+      const width = await object.getCSSProperty("width");
+      console.log(width);
+      const valueWidth = await width.value;
+      await expect(valueWidth).toEqual("150px");
+    }
+  });
+
+  it("Checking the appearance of the search button", async () => {
+    const search = await Menu.searchButton;
+
+    const fontSize = await search.getCSSProperty("font-size");
+    console.log(fontSize);
+    const valueFontSize = await fontSize.value;
+    await expect(valueFontSize).toEqual("16px");
+
+    const paddingLeft = await search.getCSSProperty("padding-left");
+    console.log(paddingLeft);
+    const valuePaddingLeft = await paddingLeft.value;
+    await expect(valuePaddingLeft).toEqual("15px");
+  });
+
+  it("Checking the appearance of the search icon", async () => {
+    const searchIcon = await Menu.searchIcon;
+
+    const backgroundPosition = await searchIcon.getCSSProperty(
+      "background-position"
+    );
+    console.log(backgroundPosition);
+    const valueBackgroundPosition = await backgroundPosition.value;
+    await expect(valueBackgroundPosition).toEqual("-290px -129px");
+
+    const width = await searchIcon.getCSSProperty("width");
+    console.log(width);
+    const valueWidth = await width.value;
+    await expect(valueWidth).toEqual("20px");
+
+    const height = await searchIcon.getCSSProperty("height");
+    console.log(height);
+    const valueHeight = await height.value;
+    await expect(valueHeight).toEqual("20px");
+  });
+
+  it("Checking the appearance of the see more buttons", async () => {
+    const seeMoreButtons = [
+      HomePage.seeMoreCuteButton,
+      HomePage.seeMoreFuunyButton,
+      HomePage.seeMoreGoodeButton,
+      HomePage.seeMoreOmgButton,
+    ];
+
+    for (let i = 0; i < seeMoreButtons.length; i++) {
+      const object = await seeMoreButtons[i];
+
+      browser.pause(1000);
+      const fontSize = await object.getCSSProperty("font-size");
+      console.log(fontSize);
+      const valueFontSie = await fontSize.value;
+      await expect(valueFontSie).toEqual("18px");
+
+      const color = await object.getCSSProperty("color");
+      console.log(color);
+      const valueColor = await color.parsed.hex;
+      await expect(valueColor).toEqual("#000000");
+    }
+  });
+
+  it("Checking the appearance of the logo", async () => {
+    const logo = await Menu.logo;
+
+    const backgroundPosition = await logo.getCSSProperty("background-position");
```

Comment:
Instead of background-position, you could check if the background image is not empty.